server/users/users.controller.ts

```diff
@@ -166,4 +166,12 @@ export class UsersController {
     public async getAll(@Query() getUsers) {
         return this.usersService.findAll(getUsers);
     }
+
+    @IsPublic()
```

Comment:
That's a privacy breach. We shouldn't allow this endpoint to be public. Are there any reasons for that?